startup/gui/layouts.py

```diff
@@ -68,8 +69,8 @@
+layouts.add( 'Standard (multi-monitor)', "GafferUI.CompoundEditor( scriptNode, _state={ 'children' : ( GafferUI.SplitContainer.Orientation.Vertical, 0.9746079163554892, ( ( GafferUI.SplitContainer.Orientation.Horizontal, 0.699764982373678, ( {'tabs': (GafferUI.GraphEditor( scriptNode ), GafferSceneUI.LightEditor( scriptNode ), GafferUI.AnimationEditor( scriptNode ), GafferSceneUI.PrimitiveInspector( scriptNode )), 'currentTab': 0, 'tabsVisible': True}, ( GafferUI.SplitContainer.Orientation.Vertical, 0.5388162951575711, ( {'tabs': (GafferUI.NodeEditor( scriptNode ), GafferSceneUI.SceneInspector( scriptNode ), GafferSceneUI.SetEditor( scriptNode )), 'currentTab': 0, 'tabsVisible': True}, {'tabs': (GafferSceneUI.HierarchyView( scriptNode ), GafferImageUI.ImageInspector( scriptNode ), GafferUI.PythonEditor( scriptNode )), 'currentTab': 0, 'tabsVisible': True} ) ) ) ), {'tabs': (GafferUI.Timeline( scriptNode ),), 'currentTab': 0, 'tabsVisible': False} ) ), 'detachedPanels' : ( { 'children' : {'tabs': (GafferUI.Viewer( scriptNode ), GafferSceneUI.UVInspector( scriptNode ), GafferDispatchUI.LocalJobs( scriptNode )), 'currentTab': 0, 'tabsVisible': True}, 'windowState' : { 'screen' : -1, 'fullScreen' : False, 'maximized' : False, 'bound' : imath.Box2f( imath.V2f( 0.104492188, 0.10969568 ), imath.V2f( 0.838867188, 0.904458582 ) ) } }, ), 'windowState' : { 'screen' : -1, 'fullScreen' : False, 'maximized' : True, 'bound' : imath.Box2f( imath.V2f( 0, 0.294409066 ), imath.V2f( 0.384375006, 0.973814607 ) ) }, 'editorState' : {'c-0-0-0-1': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-0-0-3': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-0-0-1': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-0-0-2': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-1-0-0': {'nodeSet': 'scriptNode.focusSet()'}, 'c-0-1-1-0-1': {'nodeSet': 'scriptNode.focusSet()'}, 'p-0-0-0': {'nodeSet': 'scriptNode.focusSet()'}, 'p-0-0-1': {'nodeSet': 'scriptNode.focusSet()'}} } )" )
```

Comment:
I did wonder a bit about the placement of this, and whether it made more sense to put it in the top right corner with the Scene Inspector. Though it does make sense here as the Hierarchy View is unnecessary when working with images (apart from Cryptomatte manifest previews)...

Reply:
Yeah, my initial instinct was to put it with the SceneInspector, but after thinking about it a bit I reckon it's better down with the HierarchyView. Mostly because it's more useful to use the NodeEditor in conjunction with the ImageInspector than to use the HierarchyView with it, but also just to keep some balance in the number of editors in each panel.